src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -177,6 +178,7 @@ public void addClientNote(Client target, Note clientNote) {
         requireAllNonNull(target, clientNote);
         target.addClientNote(clientNote);
         // update the TagNoteMap:
+        // todo: update address book to reflect the new note added?
```

Comment:
probably no need for this, since we're mutating the client directly

Reply:
okay, understood 👍